ecs-cli/utils/logger/logger.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2015-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package logger
+
+import "github.com/awslabs/amazon-ecr-credential-helper/ecr-login/config"
+
+func SetupLogger() {
+	config.SetupLoggerWithConfig(loggerConfig())
+}
+
+func loggerConfig() string {
+	return `
```

Comment:
It's fine if you don't want to change this in this PR, but it would be better to configure seelog programmatically instead of hard-coded string config.